### ISSUE TITLE
Implement `nvm switch` command

### ISSUE THREAD

**Issue by xavierfoucrier** [nvm-windows/nvm#884]
Just a suggestion / idea, when working with `NVM` on multiple projects **where NodeJS versions differs**, it would be nice to have a simple command to easily "switch" between current NodeJS used version and the previous one.

Implementation example:

```shell
cd ./project-a
nvm use 18

cd ../project-b
nvm use 16

cd ../project-a
nvm switch  <-- switch to NodeJS v18
```

> Note that `nvm switch` should do nothing if current/previous versions are the same, which is the case when called the first time.

**Comment by coreybutler** [comment]
Correct me if I'm misinterpreting, but it seems like the value of this proposal is not needing to know what the prior version was... i.e. less mental overhead. 

One of the planned features for [Runtime](https://github.com/coreybutler/nvm-windows/wiki/Runtime) are aliases, i.e. `nvm use myprojectalias`. Alternatively, many folks have requested `.nvmrc` support to automatically switch based on whatever is defined in the `.nvmrc` file within a project directory. I've been [steadfast about _not_ adding this to NVM4W](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#why-isnt-nvmrc-supported-why-arent-some-nvm-for-macoslinux-features-supported), but I believe there is a place for it (or similar solution) in Runtime.

Bottom line, does your vision of `switch` concept offer something one of these other solutions doesn't?

**Comment by xavierfoucrier** [comment]
Hello @coreybutler!

Thanks for your fast reply.
Of course I can confirm that it's just for less mental overhead when working on two or more projects.

I didn't know that you have planned to build an `alias` feature, looks like a good plan!
May be this could fit my needs to easily switch between Node versions.

It's just a proposal / idea that came to me when using NVM a day long.
If you think it's out of scope or useless, feel free to close it 😉

**Comment by coreybutler** [comment]
Reducing mental overhead is not useless or out of scope. I just wanted to clearly define the challenge so the best solution surfaces. Whether it will be a `switch` statement or something is something we can test in early Runtime betas. I'll leave this open for further comments.

**Comment by xavierfoucrier** [comment]
Just leaving a comment to prevent this from closing by @github-actions :wink:

**Comment by xavierfoucrier** [comment]
Just leaving a comment to prevent this from closing by @github-actions 😉

**Comment by micky2be** [comment]
For anyone interested in a pseudo solution, I just spend a couple of hours writing a [PowerShell script](https://gist.github.com/micky2be/1b9ae3116322222720ba53aa72798b33)

This is for my personal use, but feel free to give it a try

**Comment by noooonee** [comment]
I also want to have a quick way to switch multiple projects. How about store project related information in a hidden sub-folder? Link reusable resources to a local repo -- Sound like how python venv been designed.

**Comment by coreybutler** [comment]
Version 2 introduces `nvm use last`. This can be used to toggle between versions:

<img width="530" height="239" alt="Image" src="https://github.com/user-attachments/assets/b970be37-bfcc-4198-9178-80f9806003bb" />
